packages/runtime/test/lib/testUtils.ts

```diff
@@ -428,6 +428,24 @@ export async function ensurePendingRelationship(sTransportServices: TransportSer
     return (await sTransportServices.relationships.getRelationships({})).value[0];
 }
 
+export async function mutualDecompositionIfActiveRelationshipExists(sTransportServices: TransportServices, rTransportServices: TransportServices): Promise<void> {
```

Comment:
test utils should be written as imperatives (for example exchangeAndAcceptRequestByMessage directly below => use sender and recipient to exchange and accept a request by message)